### DISCUSSION TITLE
How to mark a task as done but actually "won't do"

### DISCUSSION THREAD

**Discussion by kwoot** [GothenburgBitFactory/taskwarrior#2535] [Q&A]
Is there a way to abandon a task?
As in: it will behave as if status is "done" but actually in the end it was decided to not continue but abandon this work?
Or should I add a tag "ABANDONDED" and mark task done?

**Comment by tbabej** [comment]
You can use `task delete` instead of `task done`, which will move the task into the `deleted` status. I usually combine deletion with a custom tag to track the reason.
